src/base/conversion.rs

```diff
@@ -118,12 +118,13 @@ macro_rules! impl_from_into_asref_1D(
               S: ContiguousStorage<N, $NRows, $NCols> {
             #[inline]
             fn into(self) -> [N; $SZ] {
-                unsafe {
-                    let mut res: [N; $SZ] = mem::uninitialized();
-                    ptr::copy_nonoverlapping(self.data.ptr(), &mut res[0], $SZ);
+                let mut res = mem::MaybeUninit::<[N; $SZ]>::uninit();
 
-                    res
-                }
+                unsafe { ptr::copy_nonoverlapping(self.data.ptr(), res.as_mut_ptr() as *mut N, $SZ); }
```

Comment:
Nit: Rustfmt usually does
```rust
unsafe { ptr::copy_nonoverlapping(self.data.ptr(), res.as_mut_ptr() as *mut N, $SZ) };
```
with the `}` before the `;`.

Reply:
Thanks :)